frontend/packages/dev-console/src/components/import/section/FormSection.scss

```diff
@@ -1,3 +1,3 @@
 .odc-form-section__heading {
-  margin: 0;
+  margin: 0 0 var(--pf-global--spacer--xl);
```

Comment:
do we need this for bottom?

Reply:
yes. or else there is no spacing between the heading and other components